.github/workflows/docs.yml

```diff
@@ -10,6 +10,9 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    # Disable this on our own internal PRs since it'll be run on the push to the branch.
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
I think this can be safely removed because this workflow only runs on pushes and not pull requests.